lib/assets/Html.js

```diff
@@ -11,6 +11,23 @@ var util = require('util'),
     JavaScript = require('./JavaScript'),
     AssetGraph = require('../');
 
+// Polyfill HTMLElement.matches
+var Element = jsdom.defaultLevel.HTMLElement;
+if (!Element.prototype.matches) {
+    Element.prototype.matches =
+        Element.prototype.matchesSelector ||
+        Element.prototype.mozMatchesSelector ||
+        Element.prototype.msMatchesSelector ||
+        Element.prototype.oMatchesSelector ||
+        Element.prototype.webkitMatchesSelector ||
+        function (s) {
+            var matches = (this.document || this.ownerDocument).querySelectorAll(s),
+                i = matches.length;
+            while (--i >= 0 && matches.item(i) !== this) {}
+            return i > -1;
+        };
+}
+
```

Comment:
Good news, https://github.com/assetgraph/assetgraph/pull/592 should save you from polyfilling `matches`.